ci/lint-docs.sh

```diff
@@ -3,6 +3,6 @@
 
 set -eux
 
-git ls-files | xargs codespell
+git ls-files | xargs codespell -L keypair,iam,hda
```

Comment:
Is `keypair` keyword still needed here since there are fixed?

Reply:
Yes, the nix expressions still says `keyPair`.